scenarios/kill_wf_attempt/script/loop.py

```diff
@@ -0,0 +1,7 @@
+import time
```

Comment:
The directory name `script` should be avoided since Anaconda has `script` module and conflict with it.

Reply:
Understood. I will change directory name from script to py_script.